prime-router/src/main/kotlin/fhirengine/utils/FHIRBundleHelpers.kt

```diff
@@ -27,6 +32,50 @@ import kotlin.reflect.jvm.isAccessible
  * A collection of helper functions that modify an existing FHIR bundle.
  */
 const val conditionExtensionurl = "https://reportstream.cdc.gov/fhir/StructureDefinition/reportable-condition"
+const val conditionCodeExtensionURL = "https://reportstream.cdc.gov/fhir/StructureDefinition/condition-code"
+
+private fun lookupCondition(code: Coding, metadata: Metadata): Coding? {
+    val mappingTable = metadata.findLookupTable("observation-mapping").also {
+        if (it == null) {
+            return null // TODO: better error if table is missing? log vs throw?
+        }
+    }!!
+    val condition = mappingTable.dataRowsMap.find {
+        it[ObservationMappingConstants.TEST_CODE_KEY.lowercase()] == code.code // TODO: why lower
+    }
+    return if (condition.isNullOrEmpty()) {
+        null
+    } else {
+        Coding(
+            condition[ObservationMappingConstants.CONDITION_CODE_SYSTEM_KEY.lowercase()], // TODO: why lower
```

Comment:
Couple todos here that need to get cleaned...

Looks like the lookup table [code](https://github.com/CDCgov/prime-reportstream/blob/b3c5dcaba04e29395941c1b1e6c6596b091af988/prime-router/src/main/kotlin/metadata/LookupTable.kt#L149) is doing the lower casing

Reply:
Yeah I suppose I was wondering why that is. It's causing inconsistencies that are wreaking havoc on my OCD. I kind of want to unilaterally use lower case for all column headers now. Or add a method/param to lookup table that doesn't lower case. 

Thoughts?